x-pack/legacy/plugins/siem/public/components/alerts_viewer/alerts_table.tsx

```diff
@@ -51,33 +51,35 @@ const defaultAlertsFilters: esFilters.Filter[] = [
   },
 ];
 
-export const AlertsTable = React.memo(
-  ({
-    endDate,
-    startDate,
-    pageFilters = [],
-  }: {
-    endDate: number;
-    startDate: number;
-    pageFilters?: esFilters.Filter[];
-  }) => {
-    const alertsFilter = useMemo(() => [...defaultAlertsFilters, ...pageFilters], [pageFilters]);
-    return (
-      <StatefulEventsViewer
-        pageFilters={alertsFilter}
-        defaultModel={alertsDefaultModel}
-        end={endDate}
-        id={ALERTS_TABLE_ID}
-        start={startDate}
-        timelineTypeContext={useMemo(
-          () => ({
-            documentType: i18n.ALERTS_DOCUMENT_TYPE,
-            footerText: i18n.TOTAL_COUNT_OF_ALERTS,
-            title: i18n.ALERTS_TABLE_TITLE,
-          }),
-          []
-        )}
-      />
-    );
-  }
-);
+interface Props {
+  endDate: number;
+  startDate: number;
+  pageFilters?: esFilters.Filter[];
+}
+
+const AlertsTableComponent: React.FC<Props> = ({ endDate, startDate, pageFilters = [] }) => {
```

Comment:
What is the intent of creating this intermediary `AlertsTableComponent` using `React.FC` and then wrapping as `AlertsTable` using `React.memo` vs the previous implementation of just using `React.memo`?

Reply:
`React.memo` propagates name of the wrapped component, but as we were passing an inline function to the `memo` we end up with `Anonymus (Memo)`

Reply:
Ahhh, perfect, that's what I was wondering! 🙂 

For anyone passing through, @patrykkopycinski outlines this behavior in https://github.com/elastic/siem-team/issues/485.